src/blocks/Map/__stories__/Map.mdx

```diff
@@ -16,7 +16,7 @@ import * as MapStories from './Map.stories.tsx';
 
 [`button: Button` — Button](?path=/docs/documentation-types--docs#button)
 
-[`map: Map` — Map description](?path=/story/components-map--y-map&viewMode=docs)
+[`map: Map` — Map description](?path=/docs/components-map--y-map)
```

Comment:
maybe there should be a link here?
https://preview.gravity-ui.com/page-constructor/1168/?path=/docs/components-map--docs